tests/cases/py_client/lib_test.py

```diff
@@ -616,3 +616,17 @@ def testUploadWithPath(self):
         self.assertEqual([f['name'] for f in self.model('folder').childFolders(
             parentType='folder', parent=parent,
             user=testUser, limit=0)], ['sub0', 'sub1', 'sub2'])
+
+    def testUploadFileWithDifferentName(self):
+        itemName = 'MyStash'
+        item = self.client.createItem(self.publicFolder['_id'],
+                                      itemName)
+
+        path = os.path.join(self.libTestDir, 'sub1', 'f1')
+        uploadedFile = self.client.uploadFileToItem(item['_id'],
+                                                    path,
+                                                    reference=None,
+                                                    mimeType=None,
+                                                    localFile='g1')
```

Comment:
We tend to prefer next-line indented instead of this visual indentation style. Also, we can avoid passing `reference` and `mimeType` arguments here since they are the defaults.